extras/gizmo/axis-shapes.js

```diff
@@ -106,10 +110,26 @@ class AxisShape {
         this._rotation = options.rotation ?? new Vec3();
         this._scale = options.scale ?? new Vec3(1, 1, 1);
 
+        this._disabled = options.disabled ?? false;
+
         this._layers = options.layers ?? this._layers;
 
         this._defaultColor = options.defaultColor ?? Color.BLACK;
         this._hoverColor = options.hoverColor ?? Color.WHITE;
+        this._disabledColor = options.disabledColor ?? Color.GRAY_COLOR;
+    }
+
+    set disabled(value) {
+        this._disabled = value ?? false;
+        if (value) {
+            for (let i = 0; i < this.meshInstances.length; i++) {
+                this.meshInstances[i].material = this._disabledColor;
```

Comment:
Do you need to update material going from enabled to disabled as well (currently that's skipped)?

Reply:
Also a little confusing that you're setting material to a color :)

Reply:
Oh yea true it will not be updated until hover is called 👍